pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -713,7 +816,7 @@ async def verify_attached_operators(self, *advisories: List[int], gather_depende
               help="The name of an assembly to rebase & build for. e.g. 4.9.1")
 @click.option("--name", metavar="RELEASE_NAME",
               help="release name (e.g. 4.6.42)")
-@click.option("--date", metavar="YYYY-MMM-DD",
+@click.option("--date", metavar="YYYY-MMM-DD", required=True,
```

Comment:
I think this is also a rebase miss

Reply:
[fixed](https://github.com/openshift-eng/art-tools/pull/341/commits/48b8dad80b8271a48057de8dd1b0ef7802b10e5f)